Add curses-like display to docker stats thanks to tcell.

eiffel-fl opened the pull request:
docker stats can display stats in a curses-way by running:
docker stats --tcell

If the window is resized the display adapts.

This pull request is related to enhancement #2034

GordonTheTurtle commented:
<!-- AUTOMATED:POULE:DCO-EXPLANATION -->
Please sign your commits following these rules:
https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "2034-tcell-display" git@github.com:eiffel-fl/cli.git somewhere
$ cd somewhere
$ git commit --amend -s --no-edit
$ git push -f
~~~

Amending updates the existing PR. You **DO NOT** need to open a new one.

eiffel-fl commented:
Unfortunately and like when I test on my computer the `make test-ece`failed for reasons that I do not understand.

If you have some ideas about the problem origin I will try to make them pass successfully.

eiffel-fl commented:
Hi!

Is it possible to have some feedback on my pull request?
I have some time to work on it, so I can try to modify it to pass tests successfully!

Best regards!

eiffel-fl commented:
HI.


First, I do hope all docker contributors are fine and the same for their relatives

I updated my code so it merges without conflicts with upstream master branch.
I also removed the "draft" tag.

Unfortunately, when running `make test` on my machine, the following failure showed up:
> FAIL: cli/command/trust TestSignCommandLocalFlag (0.19s)
    sign_test.go:307: assertion failed: expected error to contain "error contacting notary server: dial tcp: lookup reg-name.io", got "Error: error contacting notary server: x509: certificate is valid for *.isp.fr, isp.fr, not reg-name.io"
Where "isp" is my Internet Service Provider, so I do not think this failure is related to my code but maybe I badly configured something with docker.

With `make test-ece`, there was 21 [failures](https://framabin.org/p/?b2b8e936034031f6#fHsNcdiXotvriAkgv248+yiqz0gv4+W+kFCxt+KhwTA=).

Best regards.

eiffel-fl commented:
Hum...
I tried to clean my git history so the commit adding my feature was on top of it but it seems I did not do anything...
I will try to clean this because this will make DCO passed and maybe jenkins.

eiffel-fl commented:
Thank you for the approval!
If you want me to rebase to upstream/master do not hesitate to ask!
